app/src/features/settings/components/BillingTeam/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { BillingTeamDetails } from "./types";
+
+export const isWorkspaceMappedToBillingTeam = (workspaceId: string, billingTeams: BillingTeamDetails[]) => {
+  return billingTeams.some((team) => team.billedWorkspaces?.includes(workspaceId));
```

Comment:
shouldn't we **only** check for the current billing team that is being whose plan is being used as the user's subscription details?

Reply:
If a user is in a workspace which is mapped to a billing team then he is already part of that billing team, Also a user can be part of multiple billing teams, there is no `current` billing team as such